playbook.yml

```diff
@@ -10,3 +10,20 @@
         password: bob123
         databases: [publicdb]
         roles: "CREATEDB,NOSUPERUSER"
+      - user: charles
+        password: charles123
+        databases: [publicdb]
+        privs: "SELECT"
```

Comment:
Ah, that explains something. I'm inclined to get this in (without tag) and we can come back to "privs" in https://github.com/openmicroscopy/ansible-role-postgresql/issues/5